beast/observationmodel/noisemodel/toothpick.py

```diff
@@ -280,12 +281,12 @@ def fit_bins(self, nbins=30, completeness_mag_cut=80, progress=True):
 
         shape = nbins, len(self.filters)
 
-        self._fluxes = np.empty(shape, dtype=float)
-        self._biases = np.empty(shape, dtype=float)
-        self._sigmas = np.empty(shape, dtype=float)
-        self._compls = np.empty(shape, dtype=float)
-        self._nasts = np.empty(shape[1], dtype=int)
-        self._minmax_asts = np.empty((2, shape[1]), dtype=float)
+        self._fluxes = np.zeros(shape, dtype=float)
+        self._biases = np.zeros(shape, dtype=float)
+        self._sigmas = np.zeros(shape, dtype=float)
+        self._compls = np.zeros(shape, dtype=float)
+        self._nasts = np.zeros(shape[1], dtype=int)
+        self._minmax_asts = np.zeros((2, shape[1]), dtype=float)
```

Comment:
Will check this.  Same - I've been bitten by this before.  Much better to set them to zeros.

Ok.  There are bunch of places.  I'll create a separate issue so that this can be dealt with explicitly and not in this big PR.